db/migrate/20200709222434_create_invitations.rb

```diff
@@ -0,0 +1,11 @@
+class CreateInvitations < ActiveRecord::Migration[6.0]
+  def change
+    create_table :invitations do |t|
+      t.integer :attendee_id
+      t.integer :attended_event_id
```

Comment:
**REQUIRED**
* An index for `attended_event_id` column is added. This will make it easier to get the number of attendants of a specific event. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/foreign_key_index_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49723079/))